zeebe/logstreams/src/main/java/io/camunda/zeebe/logstreams/impl/flowcontrol/FlowControl.java

```diff
@@ -122,6 +122,25 @@ public void onProcessed(final long position) {
     cleanupUnprocessed(position);
   }
 
+  public Limit getRequestLimit() {
+    return requestLimit;
+  }
+
+  public void setRequestLimit(final Limit requestLimit){
```

Comment:
:thought_balloon: Please check that metrics are also reset accordingly. For example, the in-flight counters should reset to 0 here I think.